unstructured/utils.py

```diff
@@ -283,8 +282,9 @@ def first(it: Iterable[Any]) -> Any:
 
 
 def only(it: Iterable[Any]) -> Any:
-    """Returns the only element from a singleton iterable. Raises an error if the iterable is not a
-    singleton."""
+    """Returns the only element from a singleton iterable.
+
+    Raises an error if the iterable is not a singleton."""
```

Comment:
PEP 257 specifies last """ goes on new line in the multi-line form. There are a few more of these below.